client/client.go

```diff
@@ -56,6 +62,18 @@ func New() (*Client, error) {
 
 	client.httpClient = httpClient
 
+	sslValidationOnce.Do(func() {
+		if err := client.Ping(); err != nil {
+			if isSSLError(err) {
+				sslValidationErr = fmt.Errorf("SSL/TLS certificate error: %v\n\nThis appears to be a certificate verification issue. To resolve this securely, configure trusted SSL/TLS certificates for the server. Using the --insecure flag bypasses SSL verification and exposes you to potential security risks, such as man-in-the-middle attacks. Only use this flag in controlled environments where you fully trust the server.", err)
+			}
```

Comment:
What if the error is not an SSL error? Should we skip it? If yes, please ignore this comment.